packages/tables/resources/views/components/selection-indicator.blade.php

```diff
@@ -12,14 +12,7 @@
                 class="inline-block animate-spin w-4 h-4 mr-3 text-primary-600"
             />
 
-            <span @class(['dark:text-white' => config('tables.dark_mode')]) x-text="
-                singularText = '{{ trans_choice('tables::table.selection_indicator.selected_count', 1, ['count' => 1]) }}'
-                pluralText = '{{ trans_choice('tables::table.selection_indicator.selected_count', 2, ['count' => 2]) }}'
-
-                return (selectedRecords.length === 1) ?
-                    singularText.replace('1', selectedRecords.length) :
-                    pluralText.replace('2', selectedRecords.length)
-            "></span>
+            <span @class(['dark:text-white' => config('tables.dark_mode')]) x-text="$pluralize('{{__('tables::table.selection_indicator.selected_count')}}', selectedRecords.length, {'count': selectedRecords.length})"></span>
```

Comment:
```suggestion
            <span @class(['dark:text-white' => config('tables.dark_mode')]) x-text="$pluralize('{{__('tables::table.selection_indicator.selected_count')}}', selectedRecords.length, { count: selectedRecords.length })"></span>
```

Reply:
Couldn't merge anymore since I merged the previous suggestion, but it is added in the latest commit now.